pyproject.toml

```diff
@@ -27,7 +27,7 @@ dev-dependencies = [
 [tool.uv.sources]
 
 [project]
-requires-python = ">=3.9"
+requires-python = ">=3.12"
```

Comment:
did you check 3.13? I think there are packages that we use that will fail with it

Reply:
modified to `requires-python = "==3.12.*"`